libs/scraper/internal/useTestBrowser.ts

```
@@ -0,0 +1,31 @@
+import { cleanProfileLocks, newPage, setupScraper } from "../src/puppeteer-init";
+import { gotoPage } from "./gotoPage";
+
+setupScraper({
+  rootFolder: './.cache/test',
+  isVisible: async () => IsManualRun,
+});
+
+export function useTestBrowser() {
+  cleanProfileLocks();
+
+  let _browser: Browser|null = null;
```

ESLint

[error] Unexpected dangling '_' in '_browser'.

(no-underscore-dangle)